docs/docs/installation.md

```diff
@@ -9,6 +9,12 @@ For Home Assistant users, there is also a [custom component (aka integration)](h
 
 Note that HassOS Addons and custom components are different things. If you are already running Frigate with Docker directly, you do not need the Addon since the Addon would run another instance of Frigate.
 
+If you are running Frigate seperate to HassOS, you will need to install and configure an (https://mqtt.org/software/)[MQTT broker] such as (https://mosquitto.org/)[Mosquitto] first.
+
+Before using Frigate with a Coral accelerator, make sure that you have the edge TPU runtime installed. Click for instructions for the [USB accelerator](https://coral.ai/docs/accelerator/get-started/) or [M.2/Mini PCI-e accelerator](https://coral.ai/docs/m2/get-started/)
```

Comment:
The USB accelerator does not require any drivers on the host. The other boards do require the apex driver on the host.